armips/move/battle_eff_seq/142.s

```diff
@@ -10,6 +10,10 @@
 .create "build/move/battle_eff_seq/0_142", 0
 
 a030_142:
+    if IF_NOTEQUAL, VAR_CURRENT_MOVE, MOVE_SHED_TAIL, _BellyDrum 
+    changevar VAR_OP_SET, VAR_ADD_STATUS1, ADD_STATUS_WORK | ADD_STATUS_SHED_TAIL
+    endscript
+_BellyDrum:
```

Comment:
is there a reason to build off of belly drum?  just curious, not sure if it's for the AI or what you're thinking